stdlib/asm/crypto/fri/frie2f4.masm

```diff
@@ -262,82 +248,52 @@ end
 #!
 #! The memory referenced above is used contiguously, as follows:
 #!
-#!   [layer_ptr ... rem_ptr ... query_start_ptr ... query_end_ptr]
+#!   [query_ptr ... layer_ptr ... rem_ptr ...]
 #!
 #! This means for example that:
 #! 1. rem_ptr - 1 points to the last (alpha0, alpha1, t_depth, d_size) tuple.
-#! 2. The length of the remainder codeword is 2 * (rem_ptr - query_start_ptr).
+#! 2. layer_ptr - 1 points to the last (e0, e1, p, poe) tuple.
 #!
 #! Cycles: for domains of size 2^n where:
-#! - n is even: 12 + 6 + num_queries * (40 + num_layers * 76 + 69) + 2626
-#! - n is odd:  12 + 6 + num_queries * (40 + num_layers * 76 + 69) + 1356
+#! - n is even: 6 + 7 + num_queries * (40 + num_layers * 76 + 26)
+#! - n is odd:  6 + 7 + num_queries * (40 + num_layers * 76 + 26)
 export.verify.1
 
-    # store [query_start_ptr, query_end_ptr, layer_ptr, rem_ptr] to keep track of all queries
+    # store [query_ptr, layer_ptr, rem_ptr, g] to keep track of all queries
     # (3 cycles)
     locaddr.0
     mem_storew
```

Comment:
I just realized that we should be able to replace this with:
```
loc_storew.0
```
Won't change the number cycles - but would make the code a bit more concise.

Reply:
Makes sense.